sdk/clientcore/perf-tests.yml

```diff
@@ -0,0 +1,19 @@
+Service: clientcore
```

Comment:
Where is this file used?

Reply:
I think we use it here @weshaggard to run perf tests https://github.com/Azure/azure-sdk-for-java/blob/main/eng/common/pipelines/templates/jobs/perf.yml#L133